### DISCUSSION TITLE
Crashplan on Synology Docker through Synology Firewall

### DISCUSSION THREAD

**Discussion by faip** [jlesage/docker-crashplan-pro#428] [Q&A] [answered]
I've set up the docker container with the image from the registry.  When I go to the correct url and port, I'm asked for my email (no where to enter a password). I enter my email and click Continue.  Nothing happens.  Then after awhile, I get an error:

Unable to sign in. Can't connect to server.

I have my Synology firewall enabled and rules set up for ports 5800, 5900 as well as the associated local ports. 

I turned off the firewall and everything works as advertised.  So there is something in the container or the firewall rules that is preventing the container from being able to connect with the Crashplan servers.

Does anyone have any ideas about what I may be missing?

**Comment by jlesage** [comment, marked as answer]
The Synology forum is probably a better place to ask this question. Here is a discussion that might be interesting: https://community.synology.com/enu/forum/11/post/142221
